### ISSUE TITLE
Do something sensible if cookies are disabled

### ISSUE THREAD

**Issue by mc0e** [AndrewBelt/hack.chat#62]
Currently without cookies it just looks broken - you don't even get to see the intro text.  

It'd be nice to explain the project still, and tell the user that they need cookies enabled if that's the case.

**Comment by AndrewBelt** [comment]
Hack.chat doesn't use any cookies, but perhaps the localStorage could have a bug since I haven't tested with them disabled. Are you getting an error message in the web console?

**Comment by mc0e** [comment]
nothing that looks relevant in the console:

```
mutating the [[Prototype]] of an object will cause your code to run very slowly; instead create the object with the correct initial [[Prototype]] value using Object.create katex.min.js:1:6573
SecurityError: The operation is insecure. client.js:33:0
Use of getUserData() or setUserData() is deprecated.  Use WeakMap or element.dataset instead.
```

**Comment by AndrewBelt** [comment]
Fixed with 2ab24d9ac9. Apparently Firefox throws an error the when the `window.localStorage` variable is even _accessed_ when cookies are disabled (even when localStorage is still enabled).

**Comment by mc0e** [comment]
nice quick turn-around :-)
